### PR TITLE
`Development`: Add exam edit e2e tests

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

The exam creation and deletion where already tested during the E2E run. Tests for editing an existing exam are missing from the test suite.

### Description

This PR extends the creation tests by also testing all the other parameters like `startDate`, `maxPoints` etc. 
This PR also adds a new test, that will edit an already created exam and checks if all the new information is processed correctly. 

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2
#### Exam Mode Test
- [ ] Test

### CODE REVIEW

File: src/main/webapp/app/exam/manage/exams/exam-detail.component.html [hunk 1]
```diff
@@ -109,17 +109,17 @@ <h3>Exam Details:</h3>
                     <ng-template #moduleNumberNotSet><span jhiTranslate="artemisApp.exam.notSet"></span></ng-template>
                 </dd>
                 <dt><span jhiTranslate="artemisApp.examManagement.visibleDate">Visible from</span></dt>
-                <dd>
+                <dd id="exam-visible-date">
```
Comment (simon-hng) [inline src/main/webapp/app/exam/manage/exams/exam-detail.component.html:112]: Maybe we could prefix ids that are relevant for javascript or tests with
`js-` or `test-` e.g. `js-exam-visible-date`

**Review by simon-hng — APPROVED** [review]
code

**Review by akesfeden — APPROVED** [review]
Additions look good and tests pass on Bamboo so I approve.

File: src/main/webapp/app/exam/manage/exams/exam-detail.component.html (hunk 1 above)
Comment (heddendorp) [reply src/main/webapp/app/exam/manage/exams/exam-detail.component.html:112]: Cypress actually has an opinion here that should be followed I think
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

**Pull request merged**